### PR TITLE
fix(id/mikoroku): Fix page list + Enable ZeistManga's search filters

### LINKED ISSUE #19113: MikoRoku: Chapter images does not load and add some feature request
### Source information

MikoRoku 1.4.8

### Source language

Indonesia

### Steps to reproduce

1. Add this manga chapter 
- https://www.mikoroku.web.id/2023/11/ataerareta-skill-wo-tsukatte-kasei-de_26.html
- https://www.mikoroku.web.id/2023/11/kichiku-eiyuu-chapter-52.html
- https://www.mikoroku.web.id/2023/11/sex-fantasy-chapter-11.html
- https://www.mikoroku.web.id/2023/11/ore-no-level-up-ga-okashi-chapter-15.html
2. Read on Tachiyomi
3. No images found
4. Open WebView
5. Chapter images shown in WebView 

### Expected behavior

Chapter finds available images

### Actual behavior

Chapter images not found

### Tachiyomi version

0.14.7

### Android version

Android 11

### Feature request
- Need a filter (because of you tap an tag in this source, it Will be `Result Not Found`)
- The Author and Artist section is Unknown

### Other details

Proff :
- Tachiyomi app
Screenshot_2023-11-27-17-21-50-74_afd66b7916d19a60815f56273408b73e
Screenshot_2023-11-27-17-22-07-54_afd66b7916d19a60815f56273408b73e
Screenshot_2023-11-27-17-22-15-08_afd66b7916d19a60815f56273408b73e

- WebView (Blurred because it contains NSFW)
IMG_20231127_172634
IMG_20231127_172612
IMG_20231127_172651

### Acknowledgements

- [X] I have searched the existing issues and this is a new ticket, **NOT** a duplicate or related to another open or closed issue.
- [X] I have written a short but informative title.
- [X] I have updated the app to version **[0.14.7](https://github.com/tachiyomiorg/tachiyomi/releases/latest)**.
- [X] I have updated all installed extensions.
- [X] I have tried the [troubleshooting guide](https://tachiyomi.org/help/guides/troubleshooting/).
- [X] If this is an issue with the app itself, I should be opening an issue in the [app repository](https://github.com/tachiyomiorg/tachiyomi/issues/new/choose).
- [X] I will fill out all of the requested information in this form.

### PR DESCRIPTION
**Pull request by Claudemirovsky**
Closes #19113
Checklist:

- [ ] Updated `extVersionCode` value in `build.gradle` for individual extensions
- [x] Updated `overrideVersionCode` or `baseVersionCode` as needed for all multisrc extensions
- [x] Referenced all related issues in the PR body (e.g. "Closes #xyz")
- [x] Added the `isNsfw = true` flag in `build.gradle` when appropriate
- [x] Have not changed source names
- [ ] Have explicitly kept the `id` if a source's name or language were changed
- [x] Have tested the modifications by compiling and running the extension through Android Studio

### CODE REVIEW

**Review by TheKingTermux — COMMENTED** [review]
Can i get a some tag?
https://www.mikoroku.web.id/search/label/H4rem
https://www.mikoroku.web.id/search/label/Dark%20Fantasy

`Genre("Dark Fantasy", "Dark Fantasy"),`
`Genre("Harem", "H4rem"),`

**Comment by Claudemirovsky** [comment]
> Can i get a Harem tag? `Genre("Harem", "H4rem"),`

Yep. I haven't added it beforehand because it didn't show up in the categories filter, but it seems that their categories filter lacks a lot of existing categories in the source.

**Comment by TheKingTermux** [comment]
> Yep. I haven't added it beforehand because it didn't show up in the categories filter, but it seems that their categories filter lacks a lot of existing categories in the source.

You can relax, I'll look for it and will update my previous comment

File: multisrc/overrides/zeistmanga/mikoroku/src/MikoRoku.kt [hunk 1]
```diff
@@ -1,26 +1,85 @@
...
+        Genre("Psychological", "Psychological"),
+        Genre("Romance", "Romance"),
+        Genre("School Life", "School Life"),
+        Genre("Sci-Fi", "Sci-Fi"),
+        Genre("Seinen", "Seinen"),
+        Genre("Shounen", "Shounen"),
+        Genre("Slice of Life", "Slice of Life"),
+        Genre("Supernatural", "Supernatural"),
+        Genre("Survival", "Survival"),
+        Genre("Tragedy", "Tragedy"),
```
Comment (TheKingTermux) [inline multisrc/overrides/zeistmanga/mikoroku/src/MikoRoku.kt:55]:
```suggestion
        Genre("Action", "Action"),
        Genre("Adventure", "Adventure"),
        Genre("Comedy", "Comedy"),
        Genre("Dark Fantasy", "Dark Fantasy"),
        Genre("Drama", "Drama"),
        Genre("Fantasy", "Fantasy"),
        Genre("Harem", "H4rem"),
        Genre("Historical", "Historical"),
        Genre("Horror", "Horror"),
        Genre("Isekai", "Isekai"),
        Genre("Magic", "Magic"),
        Genre("Mecha", "Mecha"),
        Genre("Military", "Military"),
        Genre("Monsters", "Monsters"),
        Genre("Mystery", "Mystery"),
        Genre("Psychological", "Psychological"),
        Genre("Romance", "Romance"),
        Genre("School Life", "School Life"),
        Genre("Sci-Fi", "Sci-Fi"),
        Genre("Seinen", "Seinen"),
        Genre("Shounen", "Shounen"),
        Genre("Slice of Life", "Slice of Life"),
        Genre("Supernatural", "Supernatural"),
        Genre("Survival", "Survival"),
        Genre("Tragedy", "Tragedy"),
```

Comment (TheKingTermux) [reply multisrc/overrides/zeistmanga/mikoroku/src/MikoRoku.kt:55]: Missing Historical

**Comment by vetleledaal** [comment]
I tend to create a small script whenever I need to extract genres. I ran this under `/search/label/Series` from DevTools and concatenated multiple pages together, removing duplicates. Couldn't find any missing ones:
```js
copy($$("a[rel=tag][type=button]").map(el => `Genre("${el.innerText.trim()}", "${decodeURIComponent(el.href.replace(/.*\//, ''))}"),`).join("\n"))
```

The site also allows `Manga` / `Novel` / `Series`, but I would ignore those.

**Comment by TheKingTermux** [comment]
> I tend to create a small script whenever I need to extract genres. I ran this under `/search/label/Series` from DevTools and concatenated multiple pages together, removing duplicates. Couldn't find any missing ones (outside of the previously mentioned ones):
> ```js
> copy($$("a[rel=tag][type=button]").map(el => `Genre("${el.innerText.trim()}", "${decodeURIComponent(el.href.replace(/.*\//, ''))}"),`).join("\n"))
> ```
> 
> The site also allows `Manga` / `Novel` / `Series`, but I would ignore those.
> 

Thank you for the tutorial (and script ofcourse), but since I can't run the script (because my PC is broken), so I'm looking for the genres one by one (and it take time, but I'm free Soo i can do it), but this will make it easier for me In looking for a genre in the future, I am very grateful for this very useful tutorial 👍🏻

**Pull request merged**